src/luos.c

```diff
@@ -928,3 +952,27 @@ void Luos_Flush(void)
 {
     Robus_Flush();
 }
+
+/******************************************************************************
+ * @brief register a new package
+ * @param package to register
+ * @return None
+ ******************************************************************************/
+void Luos_AddPackage(void (*Init)(void), void (*Loop)(void))
+{
+    package_table[package_number].Init = Init;
+    package_table[package_number].Loop = Loop;
+
+    package_number += 1;
+}
+
+/******************************************************************************
+ * @brief Create a service to signal empty node
+ * @param None
+ * @return None
+ ******************************************************************************/
+void Luos_EmptyNode(void)
+{
+    revision_t revision = {.unmap = {1, 0, 0}};
+    Luos_CreateContainer(0, LUOS_LAST_TYPE, "empty_node", revision);
```

Comment:
Watch out, LUOS_LAST_TYPE could be used by a user type of custom services!
Perhaps we could use the VOID service available on luos_list.h